test_team1.py

```diff
@@ -27,6 +27,43 @@ def test_us_street_address(self):
         streetAddress = 'My street address is 1234 Invalid Input Drive, Los Angeles, CA, 271'
         self.assertFalse(find_us_street_address(streetAddress))
 
+    def test_credit_card_number(self):
+        # Test Valid
+        cardNum = '1234 5678 9012 3456'
+        self.assertFalse(find_credit_card_number(cardNum))
+
+        cardNum = '1234 5678 3495 3456'
+        self.assertFalse(find_credit_card_number(cardNum))
+
+        cardNum = '6543 5678 9012 3456'
+        self.assertFalse(find_credit_card_number(cardNum))
+
+        #Test invalid
+        cardNum = '123B4 5678 9012 3456'
+        self.assertFalse(find_credit_card_number(cardNum))
+
+        cardNum = '1234 5678 9012'
+        self.assertFalse(find_credit_card_number(cardNum))
+
+    def test_twiter_handle(self):
+        # Test valid
+        userName = '@MHAOfficial'
+        self.assertFalse(find_twitter_handle(userName))
```

Comment:
Same issue here, find_twitter_handle() should return True if it finds one.  Also keep in mind you should really be able to find it in the middle of a string, for example 
```
my handle is @jon_smith and yours is different
```
But be careful, you wouldn't want to match
```
my email is jon@smith.com
```

Reply:
Thanks for clearing this up for me I was not too sure about the requirements for the Twitter handles.